File: src/main/java/kr/co/vilez/appointment/model/dto/AppointmentDto.java
package kr.co.vilez.appointment.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AppointmentDto {
    private int roomId;
    private int appointmentId;
    private int boardId;
    private int shareUserId;
    private int notShareUserId;
    private String appointmentStart;
    private String appointmentEnd;
    private String title;
    private int state;
    private String date;
    private int type;
    private String status;

}

File: app/src/main/java/kr/co/vilez/data/model/AppointmentDto.kt
package kr.co.vilez.data.model

data class AppointmentDto(val boardId: Int,
                          val type: Int,
                          val title: String,
                          val shareUserId: Int,
                          val notShareUserId: Int,
                          val appointmentStart: String,
                          val appointmentEnd: String,
                          val date: String,
                          val roomId: Int)
